osu.Game/Graphics/Cursor/MenuCursor.cs

```diff
@@ -125,6 +127,14 @@ protected override void PopOut()
             activeCursor.ScaleTo(0.6f, 250, Easing.In);
         }
 
+        private static bool shouldRotate(MouseEvent e)
+        {
+            if (e.IsPressed(MouseButton.Left) || e.IsPressed(MouseButton.Right))
+                return true;
+            else
+                return false;
+        }
```

Comment:
No need to inspect the bool to then return its exact value explicitly anyway:

```suggestion
        private static bool shouldRotate(MouseEvent e) => e.IsPressed(MouseButton.Left) || e.IsPressed(MouseButton.Right);
```

(maybe rename to `shouldRotateCursor` too for a tad more readability too?)

Reply:
I didn't know you could do that. Cool!